### PR TITLE
[CHI-331] Replace modal back svg icon with chi web font icon

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/210/ for more details.

File: src/chi/components/modal/modal.scss [hunk 1]
```diff
@@ -58,23 +58,27 @@ $animation-duration: 0.5s;
...
         outline: none;
         padding: 0;
         position: absolute;
         top: 1.25rem;
       }
 
       &::before {
-        background-image: url(icon-back.svg);
-        content: '';
+        content: '\e91e';
```
Comment (mattnickles) [inline src/chi/components/modal/modal.scss]: This is not the icon for icon-chevron-left. Please replace with `\e91f`

File: src/chi/components/modal/modal.scss [hunk 2]
```diff
@@ -58,23 +58,27 @@ $animation-duration: 0.5s;
...
         display: block;
+        font-family: 'chi';
+        font-size: 1rem;
+        font-weight: normal;
         height: 1rem;
+        line-height: 1.2rem;
         margin-right: 0.5rem;
+        transform: rotate(90deg);
```
Comment (mattnickles) [inline src/chi/components/modal/modal.scss]: Remove transform

File: src/chi/components/modal/modal.scss (hunk 1 above)
Comment (davidojedacl) [reply src/chi/components/modal/modal.scss]: done

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/212/ for more details.

**Pull request merged**